app/bot.go

```diff
@@ -77,10 +77,13 @@ func (a *App) PatchBot(botUserId string, botPatch *model.BotPatch) (*model.Bot,
 	user.Username = patchedUser.Username
 	user.Email = patchedUser.Email
 	user.FirstName = patchedUser.FirstName
+	user.UpdateAt = model.GetMillis()
 	if _, err := a.Srv.Store.User().Update(user, true); err != nil {
 		return nil, err
 	}
 
+	a.sendUpdatedUserEvent(*patchedUser)
```

Comment:
Should we return the user given by update instead of faking it with modifying the update at?